tests/unit/test_x509.cpp

```diff
@@ -62,6 +62,9 @@ class X509Test : public ::testing::Test
     std::unique_ptr<X509Certificate> _root1_int1_cert1;
 
     std::unique_ptr<X509Certificate> _root2;
+    std::unique_ptr<X509Certificate> _root2_int1;
```

Comment:
I wouldn't bloat test class and SetUp() method this something which is used once in one single test-case. If you need it in one place - limit the scope and create it right in the test method. Make sense?

Reply:
Makes sense. Changed the scope of the added certificates accordingly.